locations/spiders/kiko_milano.py

```
@@ -22,3 +23,12 @@ async def start(self) -> AsyncIterator[JsonRequest]:
                 "Referer": "https://kiko-stores.kikocosmetics.com/",
             },
         )
+
+    def post_process_item(self, item, response, location):
+
+        item["opening_hours"] = OpeningHours()
+        for i in range(len(location["openingHours"])):
+            hours = location["openingHours"][i]
+            oh.add_ranges_from_string(DAYS[i] + " " + hours)
```

Flake8 (7.3.0)

[error] undefined name 'oh'

(F821)